cri/stream/runtime.go

```diff
@@ -53,6 +53,7 @@ func (s *streamRuntime) Exec(ctx context.Context, containerID string, cmd []stri
 	}
 
 	attachConfig := &mgr.AttachConfig{
+		Stdin:       streamOpts.Stdin,
```

Comment:
can we add test case to lock this error?

Reply:
Maybe we should make an issue for cri-tools to make the test more complete.